test/runtests.jl

```diff
@@ -59,11 +59,12 @@ end
     include("test_TotalCosts.jl")
     include("test_TotalForcing.jl")
     include("test_Permafrost.jl")
-    ## include("test_mcs.jl") # missing data
+    include("test_mcs.jl")
     include("test_scenarios_mcs.jl")
     include("test_scenarios.jl")
     include("test_standard_api.jl")
     ## include("test_scc.jl") # Takes very long to run.
```

Comment:
Can you also check that this passes? Actually, we should move this into `runtests_clean.jl` so it gets run any time we want to do a thorough check.

Reply:
Sure, I'll double check that.  Similar question above I guess in the spirit of `runtests.jl` and `runtests_clean.jl` ... should the difference be just whether it's cleaned out or not, or do we actually want the tests on (1) extensions and (2) scc to _only_ be in `runtests_clean.jl` which will mean it both cleans folders and is more thorough.

Doesn't matter to me, as long as the thorough one is what we use in continuous integration testing (which we do), but thought I'd check!

Reply:
For now I just moved the entire `test_mcs.jl`  to `runtests_clean.jl` so they will be tested in CI here!

Reply:
done

Reply:
I'm happy with the way you've done this for now-- checking that it runs, with the expectation that we'll add in the values soon.